src/Api/Models/Response/Organizations/OrganizationResponseModel.cs

```diff
@@ -65,6 +66,7 @@ public OrganizationResponseModel(Organization organization, string obj = "organi
     public string BusinessTaxNumber { get; set; }
     public string BillingEmail { get; set; }
     public PlanResponseModel Plan { get; set; }
```

Comment:
🤔 Should we create a tech debt ticket to update this property name to `PasswordManagerPlan`?

Reply:
Kind of. These plan objects need refactoring generally - I think we'll probably end up combining them, imo we should have an "Enterprise Plan" which contains different possible products, not a "PM plan" and an "SM plan". This is tracked in AC-1451 to be done immediately after GA. I'll update the description in Jira.